api/spec/requests/api/v1/tokens_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "swagger_helper"
+
+RSpec.describe "Tokens", type: :request do
+
+  include_context("authenticated request")
+
+  before(:each) {
```

Comment:
@ConnorChristensen you asked me how to avoid turning the `let` into `let!` — I'd use a before callback that runs before each assertion to touch the admin model.

Reply:
Nice! Thanks for the tip!